src/Microsoft.PowerShell.PSReadLine/Completion.cs

```diff
@@ -356,8 +356,8 @@ private void PossibleCompletionsImpl(CommandCompletion completions, bool menuSel
             ConsoleBufferBuilder cb;
             if (Options.ShowToolTips)
             {
-                const string seperator = "- ";
```

Comment:
PSReadLine is another module that has its own repo https://github.com/lzybkr/PSReadLine and will be replaced by git submodule or something similar